EDDI/MainWindow.xaml.cs

```diff
@@ -617,6 +619,8 @@ public static void prepareLog(IProgress<string> progress)
                         // Do nothing, adding to the debug log creates a feedback loop
                     }
                 }
+                if (outputLines.Count == 0) { Logging.Error("Error parsing log. Algorithm failed to return any matches."); return; }
```

Comment:
I'll tolerate a `{return;}` on the same line but no more.
Will push a commit to unwrap.